.github/workflows/release.yaml

```diff
@@ -4,12 +4,16 @@ on:
     # Inputs the workflow accepts.
     inputs:
       tag:
-        description: "Tag name, e.g. 0.4.0"
-        default: 0.6.0
+        description: "Tag name, e.g. 0.8.0"
+        default: 0.8.0
         required: true
       release:
-        description: "Release name, e.g. release-0.4.0"
-        default: release-0.6.0
+        description: "Release name, e.g. release-0.8.0"
+        default: release-0.8.0
+        required: true
+      replaces:
+        description: "Version to be replaced in bundle, e.g. 0.7.3"
+        default: 0.7.3
```

Comment:
Shouldn't the default be "" ?

Reply:
Agree we should change to `""`. I think the same goes for tag and release as well right?

Reply:
thinking of it, we should only need the `tag` and the `version-replace` the `release` can be computed :)